coordinator/scripts/luigi_scoop_server.py

```diff
@@ -40,7 +40,7 @@ def __init__(self):
     self.scooping_status_ = 'NULL'
 
     callback_lambda = lambda x: self.ReceiveIceCreamOrder_(x)
-    self.server_ = actionlib.SimpleActionServer('luigi_scoop', OrderIceCreamAction, self.ReceiveIceCreamOrder_, auto_start=False)
+    self.server_ = actionlib.SimpleActionServer('luigi_scoop', OrderIceCreamAction, callback_lambda, auto_start=False)
```

Comment:
how is this better? :D

Reply:
I was debugging sth and just left it, it doesn't matter
 :D